app/models/waste_exemptions_engine/registration.rb

```diff
@@ -29,5 +33,13 @@ def state
 
       "ceased"
     end
+
+    def renewable?
```

Comment:
Though minor these new methods are public. So do you think we should have some unit tests for them?

Reply:
Sure! will add some